### DISCUSSION TITLE
I can't change the resolution or bitrate when using a Raspberry Pi Camera

### DISCUSSION THREAD

**Discussion by infinitoxxx** [bluenviron/mediamtx#1929] [Questions]
Hello, I can't change the resolution or bitrate when using a Raspberry Pi Camera v3. The parameters entered in mediamtx.yml are not being passed. What am I doing wrong?

thanks

**Comment by aler9** [comment]
Hello, setting the parameters in the configuration file should be enough to change resolution or bitrate, otherwise it's a bug. If you have problems, open an issue and follow instructions in order to allow us to replicate the problem.
